src/compounds/Item/Item.jsx

```diff
@@ -92,10 +92,10 @@ Item.propTypes = {
     description: PropTypes.string,
     id: PropTypes.number.isRequired,
     img: PropTypes.shape({
-      src: PropTypes.string.isRequired,
+      src: PropTypes.string,
       alt: PropTypes.string,
-    }).isRequired,
-    name: PropTypes.string.isRequired,
+    }),
```

Comment:
why are we no longer requiring the img or the img src?

Reply:
was using the same logic from when we paired about this, they were throwing an error when the item was loading. just added them back though